📜Create texture atlas loading tutorial (#27)
KinsonDigital/VelaptorDocs: docs/tutorials/guides/adding-content.mdx

---
sidebar_position: 2
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# Adding Content

All games need content.  Content is the lifeblood of a game.  Without content, there is no game.  In this section, we will

> learn how to add content to our game.

Reviewer: Reword
```suggestion
Content is the lifeblood of a game; without it, there is no game.  In this section, we will
learn how to add content to our game.
```

Various types of content can be added to a game.  In this tutorial, we will be adding an image to our game.  This image is what
is loaded and consumed by Velaptor to be converted into a `Texture` object.  Other kinds of content can be added to a game, such as
fonts, sounds, and more.  You can even build custom content loaders to load content types for your game to use.

> 

Reviewer: Reword
```suggestion
Various types of content can be added to a game.  In this tutorial, we will be adding an image to our game which will be loaded and consumed by Velaptor and converted into a `Texture` object.  Other content such as fonts and sounds can also be added.  You can even build custom content loaders to load content types for your game.
```

> The default kinds of content supported by Velaptor are:

Reviewer: Reword
```suggestion
The default content supported by Velaptor are:
```

- PNG images
- OGG audio files
- MP3 audio files
- Atlas texture meta-data files (these are in JSON format)
- Fonts (TTF files)

Below are the steps to add content to your game and is the same process no matter what kind of content you are adding.

> The only difference is the folder location you are adding your content to based on the type of content you are adding.

Reviewer: Reword
```suggestion
Below are the steps to add content to your game.  The steps are the same, no matter what kind of content you are adding; the only difference is the folder location.
```
